spec/features/emails_spec.rb

```diff
@@ -165,60 +149,44 @@
     end
 
     scenario 'Do not send email about own poll comments' do
-      user = create(:user, email_on_comment: true)
-      poll = create(:poll, author: user)
       comment_on(poll, user)
-
-      expect { open_last_email }.to raise_error 'No email has been sent!'
+      expect { open_last_email }.to raise_error('No email has been sent!')
     end
 
     scenario 'Do not send email about poll question comment unless set in preferences' do
-      user = create(:user, email_on_comment: false)
-      poll = create(:poll, author: user)
+      user.update(email_on_comment: false)
       comment_on(poll)
-
-      expect { open_last_email }.to raise_error 'No email has been sent!'
+      expect { open_last_email }.to raise_error('No email has been sent!')
     end
   end
 
   context 'Comment replies' do
-    scenario "Send email on comment reply", :js do
-      user = create(:user, email_on_comment_reply: true)
+    let(:user) { create(:user, email_on_comment_reply: true) }
+
+    scenario 'Send email on comment reply', :js do
       reply_to(user)
 
       email = open_last_email
       expect(email).to have_subject('Someone has responded to your comment')
       expect(email).to deliver_to(user)
       expect(email).not_to have_body_text(debate_path(Comment.first.commentable))
       expect(email).to have_body_text(comment_path(Comment.last))
-      expect(email).to have_body_text(I18n.t("mailers.config.manage_email_subscriptions"))
+      expect(email).to have_body_text(I18n.t('mailers.config.manage_email_subscriptions'))
       expect(email).to have_body_text(account_path)
     end
 
-    scenario "Do not send email about own replies to own comments", :js do
-      user = create(:user, email_on_comment_reply: true)
+    scenario 'Do not send email about own replies to own comments', :js do
       reply_to(user, user)
-
-      expect { open_last_email }.to raise_error "No email has been sent!"
+      expect { open_last_email }.to raise_error('No email has been sent!')
     end
 
-    scenario "Do not send email about comment reply unless set in preferences", :js do
-      user = create(:user, email_on_comment_reply: false)
+    scenario 'Do not send email about comment reply unless set in preferences', :js do
+      user.update(email_on_comment_reply: false)
       reply_to(user)
-
-      expect { open_last_email }.to raise_error "No email has been sent!"
+      expect { open_last_email }.to raise_error('No email has been sent!')
     end
   end
 
-  scenario "Email depending on user's locale" do
```

Comment:
Its not very clear to me why this scenario is a removed... its a duplicated of which one? A mention on the commit message (https://github.com/AyuntamientoMadrid/consul/pull/1362/commits/a6aa83f0643e2350cfce1a4bbffc3a2d9bfb1a3c) would have been 👌 to clarify it

Reply:
🤔 yes the first scenario `scenario "Signup Email" do` has the same content but... I think the problem is not a duplication, by reading this removed scenario's text its clear that it's not testing what it says and maybe we could fix it instead of remove it: